symforce/opt/optimizer.h

```diff
@@ -72,34 +72,17 @@ class Optimizer {
   /**
    * Constructor that copies in factors and keys
```

Comment:
I think we should update this docstring to something like `Base constructor`, since the `copies in factors and keys` part is only useful as a distinction with the other versions below that take rvalue-references